frontend/src/views/producer/dashboard/event-financial-panel/EventFinancialPanel.jsx

```diff
@@ -0,0 +1,124 @@
+import { connect } from '../../../../utils';
+import { values } from 'mobx';
+
+const Wrapper = styled.div`
+  margin: 1rem;
+  flex-basis: 20rem;
+  margin-top: calc(3rem + 5rem);
+  background-color: white;
+  border-radius: 8px;
+  overflow: auto;
+  padding: 1rem;
+  ${props => !props.show && 'display: none;'};
+`;
+const StyledList = styled.li`
+  list-style: none;
+  ul {
+    padding: 0.5rem;
+  }
+  i {
+    color: ${props => props.color};
+    margin-right: 0.5rem;
+    font-weight: 800;
+  }
+`;
+
+export const ListItem = ({ style, iconName, title, content, color }) => (
+  <StyledList style={style} color={color}>
+    <div>
+      {iconName && <Icon type={iconName} />}
+      <Typography type="largebody" color={color}>
+        {title}
+      </Typography>
+    </div>
+    {React.isValidElement(content) ? (
+      <ul>{content}</ul>
+    ) : (
+      <Typography type="body">{content}</Typography>
+    )}
+  </StyledList>
+);
+const ListWrapper = styled.ul`
+  padding: 0;
+  padding-top: 0.5rem;
+  margin: 0;
+`;
+
+export default withTheme(
+  connect('store')(
+    class EventFinancialPanel extends React.Component {
+      render() {
+        const thisEvent = this.props.store.selectedEvent;
+        const reservationList =
+          thisEvent && this.props.store.reservationsAndOrders
+            ? values(this.props.store.reservationsAndOrders).filter(
+                r => r.eventId.id == thisEvent.id,
+              )
+            : [];
+
+        const palette = this.props.theme.palette;
+        return (
+          <Wrapper show={thisEvent}>
+            <Typography type="subheader">Reservation status</Typography>
+            <ListWrapper>
+              <ListItem
+                title="Seats left"
+                content={thisEvent && thisEvent.totalAvailableTickets}
+                color={palette.primaryDeep}
+              />
+              <ListItem
+                title="Reservations"
+                color={palette.primaryDeep}
+                content={reservationList.map(elem => {
+                  return (
+                    <div>
+                      <ListItem
+                        title={elem.name}
+                        color={
+                          elem.paymentCompleted
+                            ? palette.lightGreen
+                            : palette.primary
+                        }
+                        iconName={elem.paymentCompleted && 'check'}
+                        content={
+                          <div>
+                            <Typography type="body">{elem.phone}</Typography>
+                            <br />
+                            {elem.tickets.map(ticket => {
+                              const ticketName = elem.eventId.ticketCatalog.find(
+                                c => c.id === ticket.priceId,
+                              ).ticketDescription;
+                              return (
+                                <React.Fragment>
```

Comment:
If possible, please reduce the nesting here somehow..

eg. by putting some of the rendering into methods of the component..